libs/components/indicators/testing/src/wait/wait-harness-filters.ts

```diff
@@ -4,4 +4,9 @@ import { SkyHarnessFilters } from '@skyux/core/testing';
  * A set of criteria that can be used to filter a list of SkyAlertHarness instances.
  */
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface SkyWaitHarnessFilters extends SkyHarnessFilters {}
+export interface SkyWaitHarnessFilters extends SkyHarnessFilters {
+  /**
+   * Only find blocking or non-blocking instances created by the `SkyWaitService`.
+   */
+  globalPageWaitType?: 'blocking' | 'non-blocking';
```

Comment:
I'm not sure I love the term 'global' here. Would `servicePageWaitType` better? That describes that it was created by the service. We don't use the term 'global' anywhere else to distinguish service page waits vs component page waits.

Reply:
done